daq/forchestrator.py

```diff
@@ -43,6 +43,7 @@ def main_loop(self):
             LOGGER.error("Exception: %s", e)
             raise
 
+    # TODO: This should likely be moved into the faucet_state_collector.
```

Comment:
I feel like _handle_faucet_events still belongs in the forchestrator since it interfaces both with faucet_event_client and faucet_state_collector.

Reply:
Maybe, but it "smells" bad b/c there's way too much faucet-event-specific logic at the high level.  You're right that the event source should be decoupled from the collector, but... this kind of mess is indicative of a broken abstraction of some sort. TODO is just to properly consider it at some point.. or at least not to go around thinking that this is "as intended" :-)